changelog.md

```diff
@@ -1,3 +1,5 @@
+2024-12-06 - 6cb384e32c6b56d65cc35801cfd54ba1b3aca5fb - CardTable/Cell/DefaultCellContainer.vue, CardTable/Header/DefaultCardHeader.vue, CardTable/Table/_variables.scss, CardTable/Table/table-row.scss, List/list.scss, ListTable/CellContainer.vue, ListTable/ListTable.vue, ListTable/__tests__/__snapshots__/list-table.spec.ts.snap, ListTable/__tests__/list-table.spec.ts - Add 11px left and right padding for each cell, remove right padding of checkbox cell, make left padding of any first coloumn 5px and right align all action buttons in action class.
```

Comment:
No need to mention test files and snapshots

Reply:
I removed them